readme.md

```diff
@@ -13,6 +13,8 @@ This is going to be a code-free "think like a programmer" class. Please close yo
 
 [Display this page.](examples/button.html)
 
+AM: Even though they should know how to do this, maybe replace "display" with the terminal command required to do this.
```

Comment:
Ah, sorry, I'm using this as a prompt for myself to display it on the projector. The students will be computer-less and so shouldn't be following along in the lesson plan.